Attestation/README.md

```diff
@@ -780,54 +780,92 @@ for secret transport.  This list is almost certainly not exhaustive.
-
-> We'd like to use the EK itself, however,
-> [that is not actually possible](https://github.com/tpm2-software/tpm2-tools/issues/1883)
-> because the `activateHandle` argument to
-> [`TPM2_ActivateCredential()`](/TPM-Commands/TPM2_ActivateCredential.md)
-> requires `ADMIN` role, and on most TPMs the auth policy for the EK
-> does not provide any way to satisfy it for this usage.  Though in
-> principle this should be possible, and it would be very convenient if
-> it was.
-
-So for this approach one has to create a long-term attestation key that
-we shall call the `LTAK`, and then the server can store
-`TPM2_MakeCredential(EKpub, LTAKname, secrets)` without knowing the
-secrets.
-
-The client has to use
-[`TPM2_CreatePrimary()`](/TPM-Commands/TPM2_CreatePrimary.md) or
-[`TPM2_CreateLoaded()`](/TPM-Commands/TPM2_CreateLoaded.md) in order to
-deterministically create the same `LTAK` (again, without the `stClear`
-attribute), else if it uses
-[`TPM2_Create()`](/TPM-Commands/TPM2_Create.md) then it must store the
-key save file somewhere (possibly in the attestation server!) or make
-the key object persistent.
+are a form of limited asymmetric encryption (`TPM2_MakeCredential()`)
+and asymmetric decryption (`TPM2_ActivateCredential()`) with some
+authorization.  The details are explained
+[here](/TPM-Commands/TPM2_MakeCredential.md) and
+[here](/TPM-Commands/TPM2_ActivateCredential.md).  Basically, there are
+two TPM key objects involved:
+
+ - a transport key (typically the `EK`),
+ - and an authorization key (typically an `AK`)
+
+and the caller of `TPM2_MakeCredential()` must specify the public part
+of the transport key and the
+[name](/Intro/README.md#Cryptographic-Object-Naming) of the
+authorization key, along with a small secret to transport.  The caller
+of `TPM2_ActivateCredential()` must then provide the handles for those
+two key objects and the outputs of `TPM2_MakeCredential()` in order to
+extract the small secret.  Typically the small secret is an AES key for
+encrypting larger secrets.
+
+So if we can store the outputs of `TPM2_MakeCredential()` long-term so
+that the client can activate over multiple reboots, then we have a way
+to deliver secrets to the client.
+
+We'll discuss two ways to do this:
+
+ - use a universally well-known key -- let's call it the `WK`
+
+   Since the `WK`'s private area is not used for any cryptography in
+   `TPM2_MakeCredential()`/`TPM2_MakeCredential()`, it can be a key that
+   everyone knows.
+
+   Note that the `WK`'s public area can name arbitrary an auth policy,
+   and `TPM2_MakeCredential()` will enforce it.
+
+   E.g., the `WK` could be the all-zeros AES key.  Its policy could be
+   whatever is appropriate for the organization.  For example, the
+   policy could require that some non-resettable application PCR have
+   the value zero so that extending it can disable use of
+   `TPM2_MakeCredential()` post-boot.
+
+ - use a long-term `AK` -- let's call it the `LTAK`
```

Comment:
I had to parse the text few times to find LTAK and WK. In my mind having the abbreviation in a single place is easier. Of course, this could just be my personal preference and not a good thing.

Just perhaps 
* LTAK - Long term Attestation Key 
* WK - Well-know key
and afterwards each key's logic and purpose.

Reply:
Sure.  In IETF documents we have the acronym expansion first and the acronym in parenthesis, then afterwards only the acronym, like this:

> Then the client uses its retroencabulator manifold (REM) to blah blah blah.  The REM can be used to ...

But I'm perfectly happy to do it the other way around.